app/models/user.js

```diff
@@ -0,0 +1,33 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'users',
```

Comment:
for convention, model's name must be singular.

We like to keep models singular, since they are representing a singular instance of that type of object.